useful-scripts/publisher-tests/index.ts

```diff
@@ -0,0 +1,340 @@
+/* eslint-disable */
+
+import ZipDirectory from './utils/ZipDirectory';
+import axios from 'axios';
+import dotenv from 'dotenv';
+import { v4 } from 'uuid';
+import fs from 'fs';
+import StreamZip from 'node-stream-zip';
+import globby from 'globby';
+import rimraf from 'rimraf';
+import FormData from 'form-data';
+import Sleep from './utils/Sleep';
+import { ReleaseDataProps, SubjectArrayT } from 'types';
+
+// disable insecure warnings
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+dotenv.config();
+const { JWT_TOKEN, API_URL, TOPIC_ID } = process.env;
+
+const createPublication = async () => {
```

Comment:
This code is mostly duplicated from the importer tests? I'd pull out common functionality from the files and then importing them into both publish and importer tests. You could put both the importer-tests and publisher-tests in the same load test directory.